### PR TITLE
Referrals sender page

### PR DESCRIPTION
**Pull request by Iamsamwhoami**

### CODE REVIEW

File: Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationOnboardingIcon.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import Foundation
+import Flow
+import UIKit
+
+struct ReferralsInvitationOnboardingIcon {}
+
+extension ReferralsInvitationOnboardingIcon: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let view = Icon(icon: Asset.clock, iconWidth: 16)
```
Comment (gustafguner) [inline Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationOnboardingIcon.swift]: Did the real-time clock become a mess? 🕐

File: Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationMemberIcon.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+struct ReferralsInvitationMemberIcon {}
+
+extension ReferralsInvitationMemberIcon: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let view = UIStackView()
+        view.axis = .horizontal
+        view.alignment = .center
+        
+        let styling = UIView()
```
Comment (gustafguner) [inline Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationMemberIcon.swift]: wouldn't `container` be a better name than `styling`?

Comment (Iamsamwhoami) [reply Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationMemberIcon.swift]: Renamed it to `boxView` instead :)

**Comment by Iamsamwhoami** [comment]
@gustafguner fixed!

File: Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationOnboardingIcon.swift (hunk 1 above)
Comment (Iamsamwhoami) [reply Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationOnboardingIcon.swift]: Haha yes, wasn't really worth spending time on 😅

**Pull request merged**